scripts/packaging/examplesfolderpreupm.ps1

```diff
@@ -5,14 +5,14 @@
     Prepares the MRTK\Examples folder for UPM packaging.
 #>
 
-$examplesRoot = "$(Get-Location)"
+$packageRoot = "$(Get-Location)"

```

Comment:
There are a bunch of things in here that use Get-Location, which I think is equivalent to using PWD:

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-location?view=powershell-7

i.e. it requires the script to be run from a specific folder.

It might be better to take in a specific directory as a parameter to the script, rather than try to auto infer based on the folder you're currently in.

Reply:
i'll work on that today

Reply:
resolved :)